src/components/BaseComponents/UserPopup/UserPopup.js

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import "../../../styles/components/popup.scss";
+
+const UserPopup = ({ popup, setPopup, user }) => {
+    return (
+        <>
+            {popup ? (
+                <>
+                    <div
+                        className="popup_closer"
+                        onClick={() => setPopup(false)}
+                    ></div>
+                    <div className="popup">
+                        <div className="popup__image">
+                            <img src={user.avatar_url} alt={user.login} />
+                        </div>
+                        <div className="popup__name">{user.name}</div>
+                        <div className="popup__login">{user.login}</div>
+                        <div className="popup__bio">{user.bio}</div>
+                        <div className="popup__location">{user.location}</div>
+                        <a
+                            href={user.html_url}
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Go to Github
+                        </a>
+                    </div>
+                </>
+            ) : null}
```

Comment:
`{popup && (...) }` in this case you don't need null. It looks bad. Divide that component and `onClick={() => setPopup(false)}` dont use logic inside the JSX, create conts with callback outside.